cmd/river/go.mod

```diff
@@ -2,19 +2,20 @@ module github.com/riverqueue/river/cmd/river
 
 go 1.21.4
 
-// replace github.com/riverqueue/river => ../..
+replace github.com/riverqueue/river => ../..
```

Comment:
Have to temporarily uncomment the `replace` because the River API changed. I'll unwind this as I'm cutting a release.